data.sql

```diff
@@ -70,4 +70,62 @@ WHERE animals.name = 'Charmander' AND owners.full_name = 'Melody Pond'
+  ((SELECT id FROM species WHERE name = 'Pokemon'),
+    (SELECT id FROM vets WHERE name = 'William Tatcher')),
+  ((SELECT id FROM species WHERE name = 'Digimon'),
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez')),
+  ((SELECT id FROM species WHERE name = 'Pokemon'),
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez')),
+  ((SELECT id FROM species WHERE name = 'Digimon'),
+    (SELECT id FROM vets WHERE name = 'Jack Harkness'));
+
+INSERT INTO visits VALUES
+  ((SELECT id FROM animals WHERE name = 'Agumon'),
+    (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24'),
+  ((SELECT id FROM animals WHERE name = 'Agumon'),
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-07-22'),
+  ((SELECT id FROM animals WHERE name = 'Gabumon'),
+    (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2021-02-02'),
+  ((SELECT id FROM animals WHERE name = 'Pikachu'),
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-01-05'),
+  ((SELECT id FROM animals WHERE name = 'Pikachu'),
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-03-08'),
+  ((SELECT id FROM animals WHERE name = 'Pikachu'),
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-05-14'),
+  ((SELECT id FROM animals WHERE name = 'Devimon'),
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2021-05-04'),
+  ((SELECT id FROM animals WHERE name = 'Charmander'),
+    (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2021-02-24'),
+  ((SELECT id FROM animals WHERE name = 'Plantmon'),
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-12-21'),
+  ((SELECT id FROM animals WHERE name = 'Plantmon'),
+    (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-08-10'),
+  ((SELECT id FROM animals WHERE name = 'Plantmon'),
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2021-04-07'),
+  ((SELECT id FROM animals WHERE name = 'Squirtle'),
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2019-09-29'),
+  ((SELECT id FROM animals WHERE name = 'Angemon'),
+    (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2020-10-03'),
+  ((SELECT id FROM animals WHERE name = 'Angemon'),
+    (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2020-11-04'),
+  ((SELECT id FROM animals WHERE name = 'Boarmon'),
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-01-24'),
+  ((SELECT id FROM animals WHERE name = 'Boarmon'),
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-05-15'),
+  ((SELECT id FROM animals WHERE name = 'Boarmon'),
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-02-27'),
+  ((SELECT id FROM animals WHERE name = 'Boarmon'),
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-08-03'),
+  ((SELECT id FROM animals WHERE name = 'Blossom'),
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-05-24'),
+  ((SELECT id FROM animals WHERE name = 'Blossom'),
+    (SELECT id FROM vets WHERE name = 'William Tatcher'), '2021-01-11');
```

Comment:
- [ ] You can refer to the above way to insert the data into visits table, below is sample for one of the insertion data.

 `INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24');`